launcher/aboutProject/aboutproject_moc.cpp

```diff
@@ -83,3 +84,141 @@ void AboutProjectView::on_pushButtonBugreport_clicked()
 	QDesktopServices::openUrl(QUrl("https://github.com/vcmi/vcmi/issues"));
 }
 
+
+void AboutProjectView::on_pushInstallAdditional_clicked()
+{
+
+	QFileInfo file = QFileInfo(QFileDialog::getOpenFileName(this, tr("Select exe file of chronicle episode"), "", tr("Application (*.exe)")));
+	QDir sourceRoot = file.absoluteDir();
+
+	if(!sourceRoot.exists())
+		return;
```

Comment:
Maybe use braces?

Cf. Apple's goto bug facilitated by missing braces: https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

Reply:
Personally prefer no braces for simple, single-line blocks.

Clang also provides warning for "misleading indentation", so we'll quickly catch bugs like that goto bug.